sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/ScalaReflection.scala

```diff
@@ -179,6 +179,7 @@ trait ScalaReflection {
       case t if t <:< definitions.ShortTpe => Schema(ShortType, nullable = false)
       case t if t <:< definitions.ByteTpe => Schema(ByteType, nullable = false)
       case t if t <:< definitions.BooleanTpe => Schema(BooleanType, nullable = false)
+      case other => throw new UnsupportedOperationException(s"Schema for type $other is currently not supported")
```

Comment:
Thanks @rxin! 
What is the preferred way to split a line then? 

Just starting a new line won't cut it (still at 102 chars)
I can use a temp var, the cost is negligible (we are throwing a runtime exception, it's orders of magnitude more than a redundant assignment)

    case other => 
        val errorMessage = s"Schema for type $other is currently not supported"
        throw new UnsupportedOperationException(errorMessage)

I think this is more readable than this 

    case other => 
        throw new UnsupportedOperationException(s"Schema for type $other is " +
            "currently not supported")

right?

in any case, thanks for your patience and welcoming attitude, I know this is pretty much the lowest priority, but I truly appreciate the fact you guys help out a first timer, this really speaks a lot on the team. Not taken for granted.